spec/features/user_visits_place_spec.rb

```diff
@@ -14,6 +14,24 @@
         and_return(Place.first)
     end
 
+    xit "with no previous entry" do
+      VCR.use_cassette("existing_place") do
+        visit "/"
+
+        page.fill_in 'nav-search',
+          :with => 'Turing School of Software & Design'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.